ui/page/components/coinformat.go

```diff
@@ -22,6 +22,10 @@ var (
 )
 
 func formatBalance(gtx C, l *load.Load, amount string, mainTextSize unit.Sp, scale float32, col color.NRGBA, displayUnitText bool) D {
+	return formatBalanceFull(gtx, l, amount, mainTextSize, scale, col, 0, displayUnitText)
+}
+
+func formatBalanceFull(gtx C, l *load.Load, amount string, mainTextSize unit.Sp, scale float32, col color.NRGBA, weight font.Weight, displayUnitText bool) D {
```

Comment:
One function is fine please. Let all callers pass a font.Weight, it's no biggie. `formatBalance` and `formatBalanceFull` are not good distinguishable names.

Also, use `font.Normal` rather than zero. 0 is a magic number that will confuse future code readers. I just had to go and check now to see what "font weight = 0" means.

Also, you can remove the `scale` property and always use `defaultScale`. There's no current use case where a different scale is passed. Verify.